README.md

```diff
@@ -56,4 +83,25 @@ This plugin use following GrapesJS plugin:
 - Builder page HTML : https://github.com/artf/grapesjs-preset-webpage
 - Parser CSS : https://github.com/artf/grapesjs-parser-postcss
 
-If you need to update it, codes are in the `plguin/GrapesJsBuilderBundle/Assets/js` folder.
+If you want to update it, codes are in the `plugin/GrapesJsBuilderBundle/Assets/js` folder.
+
+-------------------------------------------------------------------
+
+## Switch back to Legacy-Builder
+
+In case you are not happy with the plugin at the moment you can easily switch back to the legacy-builder (original Mautic builder). You can do so very quickly:
```

Comment:
```suggestion
In case you are not happy with the plugin at the moment, you can easily switch back to the legacy Builder (original Mautic builder). You can do so very quickly:
```